### PR TITLE
Debug #162: Deleting projects with same names

### PR DESCRIPTION
**Pull request by SteffenBrinckmann**

### CODE REVIEW

**Comment by SteffenBrinckmann** [comment]
@jmurugan-fzj can you please review

**Comment by jmurugan-fzj** [comment]
@SteffenBrinckmann I simply tried to create a new project named "NewProject" and then immediately tried to delete the same by clicking MORE | Delete

The following error is been thrown:

image

File: pasta_eln/GUI/project.py [hunk 1]
```diff
@@ -200,6 +201,12 @@ def execute(self, command:list[Any]) -> None:
         if '-branch' in doc and len(doc['-branch'])>0 and 'path' in doc['-branch'][0]:
           oldPath = self.comm.backend.basePath/doc['-branch'][0]['path']
           newPath = self.comm.backend.basePath/('trash_'+doc['-branch'][0]['path'])
+          if newPath.exits():
```
Comment (jmurugan-fzj) [inline pasta_eln/GUI/project.py:204]: Change to newPath.exists() ?

**Review by jmurugan-fzj — CHANGES_REQUESTED** [review]

File: pasta_eln/GUI/project.py [hunk 2]
```diff
@@ -200,6 +201,12 @@ def execute(self, command:list[Any]) -> None:
...
+            ret = QMessageBox.critical(self, 'Warning', 'Old project data exists. Do you want to delete?', \
```
Comment (jmurugan-fzj) [inline pasta_eln/GUI/project.py:205]: When a project is created once and deleted, thereafter whenever I delete the project with the same name, the following popup is shown, why is this popup important to the user?

image

**Comment by SteffenBrinckmann** [comment]
@jmurugan-fzj I did the repair

**Review by jmurugan-fzj — APPROVED** [review]

**Pull request merged**